routers/users.js

```diff
@@ -135,4 +141,37 @@ userRouter.get('/', async (req, res) => {
 		return res.sendStatus(500);
 	}
 });
-module.exports = userRouter;
+
+/**
+ * @api {delete} /api/users/:id Delete a User by its Id
+ * @apiName DeleteUser
+ * @apiGroup Users
+ * @apiVersion 0.1.0
+ *
+ * @apiParam {String} id Id of the User being deleted.
+ *
+ * @apiSuccessExample {json} Success-Response:
+ * 		HTTP/1.1 200 OK
+ */
+userRouter.delete('/:id', [param('id').isMongoId()], async (req, res) => {
+	try {
+		const errors = validationResult(req);
+		if (!errors.isEmpty()) {
+			return res.status(400).json({ errors: errors.array() });
+		}
+
+		const user = await userDoc.findByIdAndRemove(req.params.id);
+
+		if (!user) {
+			return res.sendStatus(404);
+		}
+
+		res.sendStatus(200);
```

Comment:
This is a small suggestion, but it makes sense here to also use a return to show that code execution stops, like for example if the user was not found or deleted you respond with return res.sendStatus(404);